docker/luminate/.gitignore

```diff
@@ -0,0 +1 @@
+requirements.txt
```

Comment:
why to ignore requirements file?

Reply:
we do this for all images as the file is generated from the Pipfile and if for some reason the build fails we don't want the dev to checkin this file by mistake.